catroid/build.gradle

```diff
@@ -26,7 +26,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.3'
+        classpath 'com.android.tools.build:gradle:2.2.3'
```

Comment:
I am using the latest version of Android Studio and it uses Instant Run feature which requires minimum gradle version 2.2.3.